elasticsearch-core/src/main/scala/com/sumologic/elasticsearch/restlastic/dsl/QueryDsl.scala

```diff
@@ -30,7 +30,7 @@ trait QueryDsl extends DslCommons with SortDsl {
 
   trait CompoundQuery extends Query
 
-  trait Filter extends EsOperation
+  trait Filter extends Query
```

Comment:
I'm wondering if `trait FilterQuery extends Query` wound be more in sync with the current naming, but it requires more changes in our repository that uses ES client

Reply:
I wanted to avoid renaming / removing existing traits, classes, etc. to not introduce any breaking changes

Reply:
So let's leave the `Filter` name